src/RakePlus.php

```diff
@@ -531,11 +496,10 @@ public function getMinLength(): int
      */
     public function setMinLength(int $min_length): RakePlus
     {
-        if ($min_length < 0) {
-            throw new InvalidArgumentException('Minimum phrase length must be greater than or equal to 0.');
-        }
+        // @note there is no need to throw an exception in runtime to me
+        // the exception can be caught and a default value used instead
+        $this->initMinLength($min_length);
```

Comment:
I would rewrite it to something like this. This code requires one of test to be updated.
```php
    try {
        $this->initMinLength($min_length);
    } catch (InvalidArgumentException $e) {
        $this->initMinLength(0);
    }
```

Reply:
Yes I agree thanks.